README.md

```diff
@@ -64,18 +83,20 @@ pipenv install py2puml
 
 Once `py2puml` is installed at the system level, an eponymous command is available in your environment shell.
 
-For example, to create the diagram of the classes used by `py2puml`, one can use:
+For example, to create the diagram of the classes used by `py2puml`, run:
+
 ```sh
 py2puml py2puml/domain py2puml.domain
 ```
```

Comment:
my intention was to be able to make such a call:

```sh
py2puml /home/luc/repos/py2puml/py2puml/domain py2puml.domain
```

But it fails, for now.

I believe it is because of the way the Python import system works: the contents of `sys.path` and `PYTHONPATH` in such calls. When doing the aforementioned call, I believe that some logic should infer that /home/luc/repos/py2puml should be used as the root directory and add it somehow in the `sys.path` / `PYTHONPATH` in order to make the call work properly. Are you familiar with such things?

To answer your question: yes, py2puml py2puml/domain would (and should) be feasible. In another issue, this PR is already heavy :smiley: